Wikipedia/Code/UIViewController+WMFNavigationBarUtilities.swift

```diff
@@ -0,0 +1,40 @@
+import UIKit
+
+extension UIViewController {
+    
+    // TODO: make a static func?
+    public func wmf_addBottomShadow(view: UIView) {
+        // Setup extended navigation bar
+        //   Borrowed from https://developer.apple.com/library/content/samplecode/NavBar/Introduction/Intro.html
+  
+        guard let bgColor = view.backgroundColor else {
+            assertionFailure("Could not get background color of view")
+            return
+        }
+        assert(bgColor != UIColor.clear, "Background color must not be clear")
+        
+        view.shadowOffset = CGSize(width: 0, height: CGFloat(1) / UIScreen.main.scale)
```

Comment:
`UIScreen.main.scale` should be `traitCollection.displayScale` to account for presentation places other than the main screen